traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -93,6 +93,11 @@ var TableServersController = function(servers, $scope, $state, $uibModal, $windo
 			hide: true,
 			filter: "agNumberColumnFilter"
 		},
+		{
+			headerName: "Hash ID",
+			field: "xmppId",
+			hide: false
```

Comment:
might need to change this to `hide: true` so it's hidden by default. otherwise, you probably broke this UI test: https://github.com/apache/trafficcontrol/blob/master/traffic_portal/test/end_to_end/deliveryServices/delivery-services-spec.js#L117

Reply:
@mitchell852: Fixed here: https://github.com/apache/trafficcontrol/pull/4917/files